easy-pbkdf2.js

```diff
@@ -1,15 +1,15 @@
-var crypto = require("crypto"),
-	_ = require("lodash");
+var crypto = require("crypto");
 
 var EasyPbkdf2 = module.exports = function( options ) {
 	if( !( this instanceof EasyPbkdf2 ) ){
 		return new EasyPbkdf2( options );
 	}
 
-	if ( _.isPlainObject( options ) ) {
-		_.each( options, function( value, key ){
-			this[ key ] = value;
-		}, this);
+	if (options instanceof Object) {
```

Comment:
`options instanceof Object` is a poor alternative to `isPlainObject` because it will return `true` for functions, arrays, and any other non-plain object.
```js
[] instanceof Object // true
```